shapely/ops.py

```diff
@@ -179,6 +179,63 @@ def triangulate(geom, tolerance=0.0, edges=False):
     gc = geom_factory(func(geom._geom, tolerance, int(edges)))
     return [g for g in gc.geoms]
 
+
+def voronoi_diagram(geom, envelope=None, tolerance=0.0, edges=False):
+    """
+    Constructs a Voronoi Diagram [1] from the given geometry.
+    Returns a list of geometries.
+
+    Parameters
+    ----------
+        geom: the input geometry whose vertices will be used to calculate
```

Comment:
This format is generally used for numpydocs: 

```rst 
param_name : param_type, default_value/'optional' if needed
    parameter description
```

Reply:
This project is my first time using numpy docstrings, so thanks for the clarification!